packages/e2e-tests/test-applications/create-next-app/pages/index.tsx

```diff
@@ -0,0 +1,101 @@
+import Head from 'next/head';
```

Comment:
Wdyt about replacing the contents of this page with the contents of `client.tsx` and deleting `client.tsx`? Just so we conform to the standard test defined in the README.

Reply:
It was mainly to have separate client / server contexts, but we can discuss it when working on server-side tests. Moved to index -> [693c5cf](https://github.com/getsentry/sentry-javascript/pull/6669/commits/693c5cfebb884e8da6221ea53bfcd1b0b8e08313)